src/style.css

```diff
@@ -1 +1,35 @@
 /* add styles here */
+:root{
+  --main-images: (./images/star-active.svg)
+}
+
+body {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
The `body` selector is a tag name selector, which is discouraged by the checklist. However, for the `body` tag, it is commonly accepted to use the tag itself for styling since it's the document's body element. Just ensure that this is the only tag selector used besides `html`.